### ISSUE TITLE
Custom scrollbar with Tailwind?

### ISSUE THREAD

**Issue by lancerossdev** [kevinzunigacuellar/kevinzunigacuellar.com#14]
Suggestion: Can you add custom scrollbar on your site with tailwind-scrollbar from NPM? I think it will look cool than default scrollbar in Chrome and Firefox 😄

**Comment by kevinzunigacuellar** [comment]
Hello, Lance thank you for opening an issue 🤩, I've never done a custom scrollbar but I wanted to ask if you were interested to make a PR. Please do not feel in any way committed, it's alright to say no.

**Comment by lancerossdev** [comment]
Yeah, sure! I just forked the repo and I will try working on it. I hope I don't mess up anything lol 😄.

**Comment by lancerossdev** [comment]
Pull Request done 5 hours ago! Now I'm just going to wait for the approval. Closing this issue now...
